modules/ROOT/pages/cache-scope-invalidate.adoc

```diff
@@ -0,0 +1,77 @@
+= Invalidate Cache Entries
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can invalidate a cache key or a complete cache by using the Invalidate Key and Invalidate Cache components respectively.
+Invalidate cache entries when you want the Cache Scope to execute all processors to generate a response, instead of returning a cached response for the existing entry.
+
+== Invalidate a Cache Key
+
+Use the Invalidate Key component to invalidate the specified cache key in the referenced caching strategy.
+
+=== Configure the Invalidate Key Component
+
+To invalidate a cache key:
+
+. In *Anypoint Studio*, drag the *Invalidate Key* component to your flow.
+. Open the *Invalidate Key* properties.
+. Specify the *Caching Strategy* where the cache key to invalidate is stored.
+. Select the method for generating the cache key to search and invalidate.
+
+Ensure that the method selected for generating the cache key in the Invalidate Component is the same used in the referenced Caching Strategy, otherwise the cache keys might not match and the process won't work as expected.
```

Comment:
this would assume that the payload of the flow using the cache and the flow invalidating it are the same, which is not necessarily the case.